client/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
```

Comment:
```suggestion
__pycache__/
*.py[cod]
*$py.class
```

Reply:
the `server/server/__pycache__`  should not be in git.
This will avoid future addition but now that it's in you will have to `git rm` it